jenkins/e2e-image/e2e-runner.sh

```diff
@@ -0,0 +1,418 @@
+#!/bin/bash
```

Comment:
pull https://github.com/kubernetes/kubernetes/pull/31330/commits/12ea9cb1db4bdd71857fc16d3293da37fe98c29a into this?